app/src/sharedTest/java/org/oppia/app/topic/review/TopicReviewFragmentTest.kt

```diff
@@ -25,10 +32,37 @@ import javax.inject.Singleton
 @RunWith(AndroidJUnit4::class)
 class TopicReviewFragmentTest {
 
+  @get:Rule
+  var topicActivityTestRule: ActivityTestRule<TopicActivity> = ActivityTestRule(
+    TopicActivity::class.java, /* initialTouchMode= */ true, /* launchActivity= */ false
+  )
+
+  @Test
+  fun testTopicReviewFragment_loadFragment_displayReviewSkills_isSuccessfully() {
+    ActivityScenario.launch(TopicActivity::class.java).use {
+      onView(atPosition(R.id.review_skill_recycler_view, 0))
+        .check(matches(hasDescendant(withId(R.id.skill_name))))
+    }
+  }
+
+  @Test
+  fun testTopicReviewFragment_loadFragment_selectReviewSkill_opensReviewActivity() {
+    topicActivityTestRule.launchActivity(null)
+    onView(atPosition(R.id.review_skill_recycler_view, 0)).perform(click())
+    val conceptCardFragment: ConceptCardFragment? = topicActivityTestRule.activity.supportFragmentManager
+      .findFragmentByTag(TopicActivity.getConceptCardFragmentTag()) as ConceptCardFragment
+    assertTrue(conceptCardFragment != null)
```

Comment:
Prefer using Truth: ``assertThat(conceptCardFragment).isNotNull()``

Reply:
Can we also verify that the concept card is being shown for the correct skill ID? E.g. is there something within the fragment we can verify as correctly present?

Reply:
Used `assertThat` instead of `assertTrue`

Actually I left the ConceptCardTest considering that it was already covered in `ConceptCardFragmentTest` but as I am checking that again it makes sense that the contents are actually get passed what we need.

Therefore updating the test-case to check if concept-card contains the content that we expect.